packages/module/src/components/CatalogTile/__snapshots__/CatalogTile.test.tsx.snap

```diff
@@ -12,18 +12,22 @@ exports[`CatalogTile href renders properly 1`] = `
     tabindex="0"
   >
     <div
-      class="pf-c-card__title catalog-tile-pf-header"
-      id="test-href-title"
+      class="pf-c-card__title"
     >
       <div
-        class="catalog-tile-pf-title"
+        class="pf-c-card__title-text catalog-tile-pf-header"
+        id="test-href-title"
       >
-        Patternfly
-      </div>
-      <div
-        class="catalog-tile-pf-subtitle"
-      >
-        Provided by Red Hat
+        <div
+          class="catalog-tile-pf-title"
+        >
+          Patternfly
+        </div>
+        <div
+          class="catalog-tile-pf-subtitle"
+        >
+          Provided by Red Hat
+        </div>
```

Comment:
These snapshot updates were needed because of structure changes for the Card in PF. I'm not sure if the catalog-tile-pf-header class being applied slightly differently now impacts anything. If so I think upstream changes will be needed to enable the class to be applied to the pf-c-card_title div.